src/app/parcels/parcelsTable/fetchParcelTableData.ts

```diff
@@ -46,7 +46,8 @@ const getParcelsQuery = (
         query = query
             .order("packing_date", { ascending: false })
             .order("packing_slot_order")
-            .order("client_full_name");
+            .order("is_delivery", { ascending: false })
```

Comment:
I think after is_delivery you also want to sort alphabetically by collection centre name?

Reply:
Also would it be difficult and would it make sense to fix this here? https://softwiretech.atlassian.net/browse/VFB-150
Ideally this should also apply to the default sorting if it doesn't already
Think it would make the most sense to sort
Default: date -> slot -> is_delivery -> collection centre name for is_delivery == false, has congestion charge if is_delivery == true
By Method: is_delivery -> collection centre name for is_delivery == false, has congestion charge if is_delivery == true and then reverse of that if it's sorted the other way